recipe/meta.yaml

```
@@ -22,7 +22,7 @@ requirements:
     - python >=3.6
     - jinja2
     - requests
-    - requests_cache
+    - requests-cache >=0.9.6
```

```suggestion
    - requests-cache >=0.6
```
I tried a few versions and the error mentioned in #18 seems to disappear with versions starting at 0.6

---

oh sounds good, cheers muchly, @bouweandela - I was a bit miffed by having to choose such a new pin, but also lazy to try and change versions and test :grin: